lib/pdf_utilities/pdf_validator.rb

```diff
@@ -44,7 +44,7 @@ def valid_pdf?
 
     class Validator
       DEFAULT_OPTIONS = {
-        size_limit_in_bytes: 100_000_000, # 100 MB
+        size_limit_in_bytes: 104_857_600, # 100 MB
```

Comment:
Is there a reason this isnt just:
```
size_limit_in_bytes: 100.megabytes
```
Rails does have all these built in, I would think we could use that instead, or at least that is what I have seen elsewhere.

It may also be a good idea to extract these all out to a single area/setting/constant so if it needs changed it only needs done 1 place, and things dont become mismatched. That can always come later though.

Reply:
I agree with the suggestions above.